src/redis.c

```diff
@@ -121,173 +122,176 @@ struct redisServer server; /* server global state */
+    {"flushall",flushallCommand,1,"w",0,NULL,0,0,0,0,0,NULL},
+    {"sort",sortCommand,-2,"wm",0,sortGetKeys,1,1,1,0,0,NULL},
+    {"info",infoCommand,-1,"rlt",0,NULL,0,0,0,0,0,NULL},
+    {"monitor",monitorCommand,1,"ars",0,NULL,0,0,0,0,0,NULL},
+    {"ttl",ttlCommand,2,"rF",0,NULL,1,1,1,0,0,NULL},
+    {"pttl",pttlCommand,2,"rF",0,NULL,1,1,1,0,0,NULL},
+    {"persist",persistCommand,2,"wF",0,NULL,1,1,1,0,0,NULL},
+    {"slaveof",slaveofCommand,3,"ast",0,NULL,0,0,0,0,0,NULL},
+    {"role",roleCommand,1,"lst",0,NULL,0,0,0,0,0,NULL},
+    {"debug",debugCommand,-2,"as",0,NULL,0,0,0,0,0,NULL},
+    {"config",configCommand,-2,"art",0,NULL,0,0,0,0,0,NULL},
+    {"subscribe",subscribeCommand,-2,"rpslt",0,NULL,0,0,0,0,0,NULL},
+    {"unsubscribe",unsubscribeCommand,-1,"rpslt",0,NULL,0,0,0,0,0,NULL},
+    {"psubscribe",psubscribeCommand,-2,"rpslt",0,NULL,0,0,0,0,0,NULL},
+    {"punsubscribe",punsubscribeCommand,-1,"rpslt",0,NULL,0,0,0,0,0,NULL},
+    {"publish",publishCommand,3,"pltrF",0,NULL,0,0,0,0,0,NULL},
+    {"pubsub",pubsubCommand,-2,"pltrR",0,NULL,0,0,0,0,0,NULL},
+    {"watch",watchCommand,-2,"rsF",0,NULL,1,-1,1,0,0,NULL},
+    {"unwatch",unwatchCommand,1,"rsF",0,NULL,0,0,0,0,0,NULL},
+    {"cluster",clusterCommand,-2,"ar",0,NULL,0,0,0,0,0,NULL},
+    {"restore",restoreCommand,-4,"wm",0,NULL,1,1,1,0,0,NULL},
+    {"restore-asking",restoreCommand,-4,"wmk",0,NULL,1,1,1,0,0,NULL},
+    {"migrate",migrateCommand,-6,"w",0,NULL,0,0,0,0,0,NULL},
+    {"asking",askingCommand,1,"r",0,NULL,0,0,0,0,0,NULL},
+    {"readonly",readonlyCommand,1,"rF",0,NULL,0,0,0,0,0,NULL},
+    {"readwrite",readwriteCommand,1,"rF",0,NULL,0,0,0,0,0,NULL},
+    {"dump",dumpCommand,2,"r",0,NULL,1,1,1,0,0,NULL},
+    {"object",objectCommand,3,"r",0,NULL,2,2,2,0,0,NULL},
+    {"client",clientCommand,-2,"rs",0,NULL,0,0,0,0,0,NULL},
+    {"eval",evalCommand,-3,"s",0,evalGetKeys,0,0,0,0,0,NULL},
+    {"evalsha",evalShaCommand,-3,"s",0,evalGetKeys,0,0,0,0,0,NULL},
+    {"slowlog",slowlogCommand,-2,"r",0,NULL,0,0,0,0,0,NULL},
+    {"script",scriptCommand,-2,"rs",0,NULL,0,0,0,0,0,NULL},
+    {"time",timeCommand,1,"rRF",0,NULL,0,0,0,0,0,NULL},
+    {"bitop",bitopCommand,-4,"wm",0,NULL,2,-1,1,0,0,NULL},
+    {"bitcount",bitcountCommand,-2,"r",0,NULL,1,1,1,0,0,NULL},
+    {"bitpos",bitposCommand,-3,"r",0,NULL,1,1,1,0,0,NULL},
+    {"wait",waitCommand,3,"rs",0,NULL,0,0,0,0,0,NULL},
+    {"command",commandCommand,0,"rlt",0,NULL,0,0,0,0,0,NULL},
+    {"pfselftest",pfselftestCommand,1,"r",0,NULL,0,0,0,0,0,NULL},
+    {"pfadd",pfaddCommand,-2,"wmF",0,NULL,1,1,1,0,0,NULL},
+    {"pfcount",pfcountCommand,-2,"r",0,NULL,1,-1,1,0,0,NULL},
+    {"pfmerge",pfmergeCommand,-2,"wm",0,NULL,1,-1,1,0,0,NULL},
+    {"pfdebug",pfdebugCommand,-3,"w",0,NULL,0,0,0,0,0,NULL},
+    {"latency",latencyCommand,-2,"arslt",0,NULL,0,0,0,0,0,NULL}
 };
 
 struct evictionPoolEntry *evictionPoolAlloc(void);
 
+const double CMDTIME_PERCENTILES[CMDTIME_PERCENTILES_COUNT] = {.25, .50, .90, .99, .999};
```

Comment:
Since the size has been specified, `CMDTIME_PERCENTILES` will have `CMDTIME_PERCENTILES_COUNT` elements (zero-filled if insufficient elements are specified, possibly generating a compiler error if too many). By omitting the size, the compiler will figure it out (and the `CASSERT` below be enforce).